alpine.Dockerfile

```
@@ -104,6 +104,14 @@ COPY --link caddy caddy
 COPY --link internal internal
 COPY --link testdata testdata
 
+# install fswatch (necessary for file watching)
+ARG FSWATCH_VERSION='1.17.1'
+WORKDIR /usr/local/src/fswatch
+RUN curl -L https://github.com/emcrisostomo/fswatch/releases/download/$FSWATCH_VERSION/fswatch-$FSWATCH_VERSION.tar.gz > fswatch.tar.gz  && \
+    tar xzf fswatch.tar.gz
+WORKDIR /usr/local/src/fswatch/fswatch-$FSWATCH_VERSION
+RUN	./configure && make && make install && ldconfig /usr/local/lib && fswatch --version
```

```suggestion
RUN ./configure && \
	make -j"$(nproc)" && \
	make install && \
	ldconfig /usr/local/lib && \
	fswatch --version
```

Good idea to add `fswatch --version`, we should do it in other images too!